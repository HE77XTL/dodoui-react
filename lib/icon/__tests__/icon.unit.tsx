import React from 'react'
import Icon from '../icon'
import * as renderer from 'react-test-renderer'
import {describe, expect, it} from '@jest/globals'

function add(a: number, b: number) {
    return a + b
}

describe('我的第一个测试用例', () => {
    it('add(1,2) 等于3', () => {
        expect(add(1,2)).toEqual(3);
    })
});

describe('icon', () => {
    it('是个 div', () => {
        const json = renderer.create(<Icon/>).toJSON();
        expect(json).toMatchSnapshot()
    })
});
