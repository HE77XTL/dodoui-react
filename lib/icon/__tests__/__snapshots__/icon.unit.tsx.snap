// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon 是个 div 1`] = `
<div>
  <svg
    className="dodo-icon"
  >
    <use
      xlinkHref="#qq"
    />
  </svg>
</div>
`;
